test/test_prototype_builtin_datasets.py

```diff
@@ -180,3 +180,27 @@ def test_label_matches_path(self, test_home, dataset_mock, config):
         for sample in dataset:
             label_from_path = int(Path(sample["path"]).parent.name)
             assert sample["label"] == label_from_path
+
+
+@parametrize_dataset_mocks(DATASET_MOCKS["usps"])
+class TestUSPS:
+    def test_label_matches_path(self, test_home, dataset_mock, config):
+        dataset_mock.prepare(test_home, config)
+
+        dataset = datasets.load(dataset_mock.name, **config)
+
+        for sample in dataset:
+            # check if correct keys exist
```

Comment:
Could you remove these and the following checks? They are just stating what the, objectively simple, code does and thus are redundant. As a rule of thumb, a code comment should state the intent rather than the content.